components/ModEditSubmissionTopbar.vue

```diff
@@ -76,7 +76,7 @@ const copySubmissionId = async () => {
 }
 
 const saveAndExit = () => {
-    moderationSubmissionStore.setUpdatingMutationFromTopBar(true)
+    moderationSubmissionStore.setUpdatingMutationFromTopBar(!moderationSubmissionStore.updatingMutationFromTopBar)
```

Comment:
Passing a state into its own set method doesn't really make sense 

Might be good to add a comment explain this and see if there's a better way to code this

Reply:
I think I was stuck in React mode since it is common to do `prev => !prev`

Reply:
Updated

Reply:
I didn't add a comment as I am not passing the state into its own set method anymore. just the boolean itself again. So I do not think a comment would be necessary anymore